### PR TITLE
Lab1. Vladimir Markov

### PR DESCRIPTION
**Pull request by markovvn1**
Completed the main task in full, as well as the bonus task

### CODE REVIEW

**Review by AzazKamaz — APPROVED** [review]

**Review by AzazKamaz — COMMENTED** [review]
Looks overengineered)

File: app_php/site/styles.css [hunk 1]
```diff
@@ -0,0 +1,27 @@
+body {
```
Comment (igooor-bb) [inline app_php/site/styles.css:1]: Maybe it is worth using a fixed number of spaces instead of tabs

File: app_php/site/styles.css [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	align-items: center;
+	margin: auto;
+}
+
+a {
+	width: 256px;
+	height: 256px;
+	background: url('/static/rabbit.svg');
+	background-size: 256px;
+}
```
Comment (igooor-bb) [inline app_php/site/styles.css:27]: And a best practice is to have an empty line at the end of the file

**Review by glebosotov — COMMENTED** [review]
Impressive

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**